src/game/GameManager.cpp

```diff
@@ -165,14 +171,14 @@ bool GameManager::createMarine(const float x, const float y) {
 }
 
 void GameManager::createMarine(const int32_t id) {
-    SDL_Rect temp = {INITVAL, INITVAL, MARINE_WIDTH, MARINE_HEIGHT};
+    SDL_Rect temp = {INITVAL, INITVAL, DEFAULT_SIZE, DEFAULT_SIZE};
```

Comment:
Why was this changed back?
All other marines are created with the MARINE globals